Data/Tree.hs

```diff
@@ -21,20 +21,38 @@
 -- Maintainer  :  libraries@haskell.org
 -- Portability :  portable
 --
--- Multi-way trees (/aka/ rose trees) and forests.
+-- = Multi-way (aka. Rose Trees) and Forests
```

Comment:
Please don't use the abbreviation "AKA" anywhere in the documentation of any package I (co-)maintain. Documentation is *formal writing*. That doesn't mean there can't be occasional, tasteful, and *actually funny* humor occasionally, but it does mean no language like that.

Reply:
Fair enough. It was in there before, I wasn't sure if you or someone else added it, I've removed it now.